easybuild/easyconfigs/r/R/R-3.5.1-foss-2018b.eb

```diff
@@ -1860,6 +1860,9 @@ exts_list = [
     ('gee', '4.13-19', {
         'checksums': ['ebd3eb754b338dc8d89a35fff149af57c1be3aa2eb6162912c7cc52a8572c292'],
     }),
+    ('pracma', '2.1.4', {
```

Comment:
@smoors This extension is already included (same version), see line 1292 in this easyconfig.